packages/nimble-components/src/card/specs/README.md

```diff
@@ -0,0 +1,111 @@
+the `nimble-card` component:
+
+```
+<nimble-card>
+    <div>
+        <h2>Heading</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        <nimble-button>Button</$nimble-button>
+    </div>
+</nimble-card>
+```
+
+While we don't have a finalized visual design spec, we expect the style will include a background color and a
+border, potentially with rounded corners. We will add design tokens for the background and border. The GitHub
+issue mentions several use cases for a card component that might call for different visual treatments, but we
+are only solving for the Routines use case at this time (grouping related configuration in a slide-out). To
+support other use cases in the future, we could potentially add appearance variants or make use of the `elevation`
+property of the component.
+
+### API
+
+[`fast-card` spec](https://github.com/microsoft/fast/blob/b78c921ec4e49ec9d7ec980f079ec114045df42e/packages/web-components/fast-foundation/src/card/card.spec.md)
+
+-   _Component Name_
+    -   `nimble-card`
+-   _Properties/Attributes_
+    -   _(none)_
+-   _Methods_
+    -   _(none)_
+-   _Events_
+    -   _(none)_
+-   _CSS Classes and Custom Properties that affect the component_
+    -   width
+    -   height
+    -   elevation
+        -   We don't currently plan to provide different styles based on elevation. This could be added in the future.
+-   _Slots_
+    -   `(default)`
+        -   Unchanged. Arbitrary content to be displayed in the `nimble-card``.
+
+### Angular integration
+
+We will create an Angular `nimble-card` directive for the component. The `nimble-card` does not require form integration, so there
+is no need for a ControlValueAccessor.
+
+### Blazor integration
+
+We will create a Blazor wrapper the component and define a `NimbleCard` class. The `nimble-card` component supports child content,
+so it will need a `ChildContent` parameter.
+
+### Additional requirements
+
+-   _User interaction: Do the FAST component's behaviors match the visual design spec? When they differ, which approach is preferable and why?_
+    -   No additional requirements. The initial `nimble-card` will have no interactions.
+-   _Styling: Does FAST provide APIs to achieve the styling in the visual design spec?_
+    -   No additional requirements expected here.
+-   _Testing: Is FAST's coverage sufficient? Should we write any tests beyond Chromatic visual tests?_
+    -   No additional requirements
+-   _Documentation: Any requirements besides standard Storybook docs and updating the Example Client App demo?_
+    -   No additional requirements
+-   _Tooling: Any new tools, updates to tools, code generation, etc?_
+    -   No additional requirements
+-   _Accessibility: keyboard navigation/focus, form input, use with assistive technology, etc._
+    -   We will set the ARIA role to `region`, based on the intended usage of the component. It may make sense to have clients set
```

Comment:
A few thoughts here on the ARIA roles:

- It doesn't look like the [fast-card](https://github.com/microsoft/fast/blob/b78c921ec4e49ec9d7ec980f079ec114045df42e/packages/web-components/fast-foundation/src/card/card.template.ts) or [mat-card](https://material.angular.io/components/card/examples) apply a role to their elements. What's the rationale for deciding that the nimble-card should have a role?
- Since the fast-card doesn't apply an ARIA role, if nimble is going to automatically set a role on the component, we'll need to fork their template for this component. That should be called out in this document (assuming I'm understanding your proposal correctly).
- If we use the `region` role, then it is also required for the element to have a label. We should make sure it's easy to for clients to set this label and avoid having accessibility issues within their apps.
- Do you have any thoughts on when a client would need to use a different role (such as `group` as you mentioned) rather than `region`? We've tried to avoid making clients apply ARIA roles on our components, so we should try to understand if there really is a use case for different roles. If there are, we should see if there is a way to encapsulate that within the nimble component.

Reply:
I hadn't thought about the necessity of forking the template to set the role, but that makes sense, I see why that would be necessary.

I got the idea of setting the role from the `mat-card` documentation here: https://material.angular.io/components/card/overview#group-region-and-landmarks

> There are several ARIA roles that communicate that a portion of the UI represents some semantically meaningful whole. Depending on what the content of the card means to your application, you can apply one of [role="group"](https://www.w3.org/TR/wai-aria/#group), [role="region"](https://www.w3.org/TR/wai-aria/#region), or [one of the landmark roles](https://www.w3.org/TR/wai-aria/#landmark) to the <mat-card> element.
>
>You do not need to apply a role when using a card as a purely decorative container that does not convey a meaningful grouping of related content for a single subject. In these cases, the content of the card should follow standard practices for document content.

The reasoning there seemed like it would apply to our `card` usage as well.

Reading more into [group](https://www.w3.org/TR/wai-aria/#group), though, I actually can't think of a case where it would make sense to have a card to use that role. Also looking at [region](https://www.w3.org/TR/wai-aria/#region), these containers may not rise to the level of "sufficiently important that users will likely want to be able to navigate to the section easily and to have it listed in a summary of the page".

Reply:
I'll defer to your expertise here. Just semantically, it makes sense to my mind that a container for grouping related configuration would be a "region", but I also think it would not be worth forking the fast-card template just for the sake of setting the role, when they made a determination that the card did not need a role.

Reply:
I agree with your assessment that I'm not sure of a case where we'd want to set the role to `group`. It's also hard to say whether or not usages of the card will be "sufficiently important". I personally lean towards just doing what the fast-card is doing unless we have a compelling reason to diverge.

Reply:
That's reasonable. I removed the sentence about setting a role. We'll follow the fast-card's example and leave the nimble-card without a role.